inc/widgets/post-loop-helper.php

```diff
@@ -17,10 +17,10 @@ function __construct( $templates ) {
 		$template_options = array();
 		if( ! empty( $templates ) ) {
 			foreach( $templates as $template ) {
-				$headers = get_file_data( locate_template( $template ), array(
```

Comment:
Are we using `locate_template` somewhere else? We're using this to allow child themes to override templates, so it's fairly important.

Reply:
The `SiteOrigin_Panels_Widgets_PostLoop_Helper` class is being instantiated with an array of the templates which are retrieved by the `get_loop_templates` function, which looks like it should load both parent and child theme templates:
https://github.com/siteorigin/siteorigin-panels/blob/900a83a19b6904104a864a2a0394cbb065322f93/inc/widgets/post-loop.php#L365-L366

Reply:
Looking at `locate_template` [here](https://core.trac.wordpress.org/browser/tags/4.9.8/src/wp-includes/template.php#L629), it looks like it's loading from `ABSPATH . WPINC . '/theme-compat/'` too. So what we have is mostly compatible, but not exactly.

It's minor though and shouldn't cause any issues. As long as we've checked this with themes and child themes, we should be good.